python/cudf/cudf/dataframe/datetime.py

```diff
@@ -255,6 +257,24 @@ def is_monotonic_decreasing(self):
             ).all()
         return self._is_monotonic_decreasing
 
+    def isin(self, values):
+        from cudf.dataframe import numerical
+        mask = None
+
+        if isinstance(values, (list, np.ndarray,)):
+            values = DatetimeColumn.from_numpy(
+                np.asarray(values)).as_numerical.data.mem
```

Comment:
you can't assume the values are coming from the host side here. I'd use `columnops.as_column` instead.